src/Lexer.js

```diff
@@ -267,7 +274,7 @@ module.exports = class Lexer {
         case 'text':
         case 'heading': {
           token.tokens = [];
-          this.inlineTokens(token.text, token.tokens);
+          this.inlineTokens(token.text, token.tokens, undefined, undefined);
```

Comment:
Isn't passing `undefined` is the same thing as not passing anything?

Reply:
Done.